Sources/Utility/Git.swift

```diff
@@ -57,7 +57,6 @@ public class Git {
         return knownVersions
     }
 
-    public class var tool: String {
-        return "git"
-    }
+    /// A shell command to run for Git. Can be either a name or a path.
+    public static var tool: String = "git"
```

Comment:
Not too strongly. I suppose if someone is going to override this, its unlikely they'll stop the undo the override at some later stage.